README.md

```diff
@@ -34,15 +34,44 @@ Rafal Jozefowicz, Oriol Vinyals, Mike Schuster, Noam Shazeer: “Exploring the L
 * `docker`: The [Docker](https://www.docker.com/) command-line interface. Follow the [installation instructions](https://docs.docker.com/install/) for your system.
 * The minimum recommended resources for this model is 8 GB Memory and 4 CPUs.
 
-## Steps
+# Steps
+
+1. [Deploy from Docker Hub](#deploy-from-docker-hub)
+2. [Deploy on Kubernetes](#deploy-on-kubernetes)
+3. [Run Locally](#run-locally)
+
+## Deploy from Docker Hub
+
+To run the docker image, which automatically starts the model serving API, run:
+
+```
+$ docker run -it -p 5000:5000 codait/max-news-text-generator
+```
+
+This will pull a pre-built image from Docker Hub (or use an existing image if already cached locally) and run it.
+If you'd rather checkout and build the model locally you can follow the [run locally](#run-locally) steps below.
+
+## Deploy on Kubernetes
+
+You can also deploy the model on Kubernetes using the latest docker image on Docker Hub.
+
+On your Kubernetes cluster, run the following commands:
+
+```
+$ kubectl apply -f https://raw.githubusercontent.com/IBM/MAX-News-Text-Generator/master/max-news-text-generator.yaml
```

Comment:
This file does not exist

Reply:
Need to wait for #5 to get merged in

Reply:
nice catch, I thought I'd already merged that, merged now